File: src/main/java/org/dukcode/ps/codetree/trail2/lesson2/IntroSumFrom1ToACertainNumber.java
package org.dukcode.ps.codetree.trail2.lesson2;

import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;

/**
 * @see <a
 * href="https://www.codetree.ai/ko/trails/complete/curated-cards/intro-sum-from-1-to-a-certain-number/description">LINK</a>
 */
public class IntroSumFrom1ToACertainNumber {

  private static BufferedReader br;
  private static BufferedWriter bw;

  public static void main(String[] args) throws IOException {
    br = new BufferedReader(new InputStreamReader(System.in));
    bw = new BufferedWriter(new OutputStreamWriter(System.out));

    int n = Integer.parseInt(br.readLine());

    bw.write(String.valueOf(solve(n)));
    br.close();
    bw.close();

  }

  private static int solve(int n) {
    int sum = 0;
    for (int i = 1; i <= n; i++) {
      sum += i;
    }

    return sum / 10;
  }
}

File: src/main/kotlin/org/dukcode/ps/codetree/trail02/chapter01/lesson02/IntroSumFrom1ToACertainNumber.kt
package org.dukcode.ps.codetree.trail02.chapter01.lesson02

import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

private val br = BufferedReader(InputStreamReader(System.`in`))
private val bw = BufferedWriter(OutputStreamWriter(System.out))

/**
 * @see <a href="https://www.codetree.ai/ko/trails/complete/curated-cards/intro-sum-from-1-to-a-certain-number/description">LINK</a>
 */
fun main() {
    val n = br.readLine().toInt()

    var sum = 0
    for (num in 1..n) {
        sum += num
    }

    val result = sum / 10

    bw.write(result.toString())

    bw.flush()
}
